jena-db/jena-tdb2/src/test/java/org/apache/jena/tdb2/graph/TestPrefixMappingTDB2.java

```diff
@@ -29,7 +29,7 @@
 import org.junit.Before;
 import org.junit.BeforeClass;
 
-public class TestPrefixMappingTDB extends AbstractTestPrefixMappingView
+public class TestPrefixMappingTDB2 extends AbstractTestPrefixMappingView
```

Comment:
:+1: Good for logging/quickly finding classes in the IDE, without having to look at the package name, great improvement!